contrib/depends/toolchain.cmake.in

```diff
@@ -136,11 +136,21 @@ endif()
 if(ARCHITECTURE STREQUAL "i686")
   SET(ARCH_ID "i386")
   if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    set(BUILD_TAG "linux-x86")
     SET(LINUX_32 ON)
+  elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    set(BUILD_TAG "win-x32")
   endif()
 endif()
 
 if(ARCHITECTURE STREQUAL "x86_64")
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    set(BUILD_TAG "linux-x64")
+  elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+    set(BUILD_TAG "freebsd-x64")
+  elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    set(BUILD_TAG "win-x64")
+  endif()
```

Comment:
Is a case for MacOSX needed here too?

Reply:
The mac one is set in the existing Darwin clause already.